pyproject.toml

```diff
@@ -86,10 +86,9 @@ all = [
     "beautifulsoup4>=4.9.3", # imposed by pandas==2.0
     "html5lib>=1.1",
     "bleach>=3.2.1",
-    "pandas>=2.0",
-    "sortedcontainers>=1.5.7", # (older versions may work)
-    "pytz>=2016.10", # (older versions may work)
-    "jplephem>=2.6", # (older versions may work)
+    "sortedcontainers>=2.1.0",  # imposed by testing with hypothesis
+    "pytz>=2020.1",  # imposed by pandas==2.0
+    "jplephem>=2.11",  # imposed by testing with skyfield==1.20
```

Comment:
I did notice that our oldest dependencies CI job does install `jplephem==2.6` like our `pyproject.toml` specifies, even though that is incompatible with our `skyfield` requirement. The reason for that is that we don't seem to be using our `test_all` dependencies anywhere in our CI, so the tests that require `skyfield` are not being executed at all. That does sound like a problem to me, but that rabbit hole is going too deep for this pull request.

Reply:
Turns out it was advantageous to update the `jplephem` requirement to `>=2.15`, which has been done in a separate pull request, so we don't have to update that requirement in this pull request anymore.